about/pricing.html.markerb

```diff
@@ -203,19 +203,30 @@ We use Lets Encrypt to issue certificates, and donate half of our SSL fees to th
 
 Inbound data transfer is free.
 
-Data transfer from apps without an assigned IP address is free. [Flycast IPs](/docs/networking/private-networking/#flycast-private-fly-proxy-services) are considered an IP assignment, so apps using Flycast will accrue data transfer costs.
+We bill for data leaving your app destined for the public internet or for apps or Machines in other regions.
 
-Otherwise, Fly.io bills per-region pricing for outbound data transfer.
+You're billed for data transfer to the public Internet and between regions on your private network:
 
-<%= partial("shared/network_pricing") %>
+- data egress to the Internet, from Machine to edge server to Internet
+- data transfer over private network between regions, from Machine to edge server and edge server to Machine
+- data transfer to some [extensions](https://fly.io/docs/about/pricing/#extensions) like Supabase or Upstash Redis (excludes Tigris Object Storage)
 
-Any traffic exiting a Fly.io Machine is considered outbound data transfer, including:
+You're not billed for the following types of traffic:
 
-* traffic destined for the internet
-* traffic destined for other Machines or apps in your network
-* traffic destined for [extensions](#extensions) like Supabase or Upstash Redis
+- all inbound data transfer
+- data transfer between apps or Machines in the same region
```

Comment:
@gargakshit do we need another sentence here like "(for orgs created after July 17 2024)"?

Reply:
I guess that would make sense.

```suggestion
- data transfer between apps or Machines in the same region (for organizations created after July 17 2024)
```